src/App.css

```diff
@@ -14,4 +14,52 @@
     background: linear-gradient(180deg, #08AEEA 0%, #2AF598 100%);
     -webkit-background-clip: text;
     -webkit-text-fill-color: transparent;
-  }
\ No newline at end of file
+}
+
+.styles_chip-input__3aumB .styles_chip__i-PT7 {
```

Comment:
why are your classnames like __3aumb. Are these system/library generated or you wrote these on your own?. If you wrote them then please change them as these are not easily understandable